src/Common/tests/Collections/ICollectionTest.cs

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using Xunit;
+using Xunit.Sdk;
+
+// ReSharper disable InconsistentNaming
```

Comment:
I haven't seen ReSharper-specific comments in corefx yet.
Personally I prefer to use ReSharper and this LGTM, but I think it should be discussed first.